pkg/controller/persistentvolume/persistentvolume_claim_binder_controller.go

```diff
@@ -343,6 +394,14 @@ func syncClaim(volumeIndex *persistentVolumeOrderedIndex, binderClient binderCli
 	return nil
 }
 
+func isBeingProvisioned(volume *api.PersistentVolume) bool {
+	value, found := volume.Annotations[pvProvisioningRequired]
```

Comment:
`isAnnotationMatch` checks that an annotation *has* given value, while `isBeingProvisioned` checks, that it *has not*  the value. Who knows what values the annotation can have (or will have in future).